py/torch_tensorrt/dynamo/lowering/_decompositions.py

```diff
@@ -172,6 +172,8 @@ def empty_permuted_decomposition(*args, **kwargs) -> torch.Tensor:
     perm = [0] * len(empty_size)
     for permute_index, permute_element in enumerate(empty_permute):
         perm[permute_element] = permute_index
+    default_device = torch.device("cuda" if torch.cuda.is_available() else "cpu")
```

Comment:
use the default device defined in our `_defaults.py` : https://github.com/pytorch/TensorRT/blob/main/py/torch_tensorrt/dynamo/_defaults.py#L37
and convert this into a torch device via `_enums.to()`